### PR TITLE
Use USAFacts data for inference instead of JHUCSSE

### PR DESCRIPTION
**Pull request by kkintaro**

### CODE REVIEW

File: R/scripts/filter_MC.R [hunk 1]
```diff
@@ -70,53 +70,35 @@ if(is.na(opt$simulations_per_slot)){
...
-data_dir <- gsub('[/][^/]*','',data_path)
+data_dir <- dirname(data_path)
 if(!dir.exists(data_dir)){
   suppressWarnings(dir.create(data_dir,recursive=TRUE))
 }
 if(!dir.exists(paste('importation',config$spatial_setup$setup_name,'case_data',sep='/'))){
   suppressWarnings(dir.create(paste('importation',config$spatial_setup$setup_name,'case_data',sep='/'),recursive=TRUE))
 }
-# Parse jhucsse using covidImportation
+# Parse USAFacts data using covidImportation
```
Comment (eclee25) [inline R/scripts/filter_MC.R:80]: using covidcommon?

Comment (kkintaro) [reply R/scripts/filter_MC.R:80]: Yup

File: R/pkgs/covidcommon/R/USAFacts.R [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+##'
+##' @param filename where case data will be stored
+##' @param url URL to CSV on USAFacts website
+##' @return data frame
+##'
+##' @importFrom dplyr select rename filter mutate
+##' @importFrom lubridate mdy
+##' @importFrom readr read_csv col_character
+##' @importFrom tidyr pivot_longer
+##'
```
Comment (eclee25) [inline R/pkgs/covidcommon/R/USAFacts.R:14]: should this be exported? or are functions still not exported in covidcommon?

Comment (kkintaro) [reply R/pkgs/covidcommon/R/USAFacts.R:14]: This is more of an internal helper function. So I didn't export it. Just get_USAFacts_data()

File: R/pkgs/covidcommon/NAMESPACE [hunk 3]
```diff
@@ -3,4 +3,13 @@
 export(as_density_distribution)
 export(as_evaled_expression)
 export(as_random_distribution)
+export(get_USAFacts_data)
 export(load_config)
+importFrom(dplyr,filter)
```
Comment (eclee25) [inline R/pkgs/covidcommon/NAMESPACE:8]: magrittr needs to be loaded to get the tidyverse pipe

File: R/pkgs/covidcommon/R/USAFacts.R [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+##' Downloads the USAFacts case and death count data
+##'
+##' @param filename where case data will be stored
+##' @param url URL to CSV on USAFacts website
+##' @return data frame
+##'
+##' @importFrom dplyr select rename filter mutate
+##' @importFrom lubridate mdy
+##' @importFrom readr read_csv col_character
+##' @importFrom tidyr pivot_longer
```
Comment (eclee25) [inline R/pkgs/covidcommon/R/USAFacts.R:13]: need to add `@importFrom magrittr %>%`

**Pull request merged**